tms_objects2.erb

```diff
@@ -730,6 +730,21 @@ xsi:schemaLocation="http://www.lido-schema.org http://www.lido-schema.org/schema
             </lido:relatedWorkRelType>
           </lido:relatedWorkSet>
         <% end %>   <%# BIBcitations |bc| %>
+        <% if TextTypeID == 305 %>
+          <lido:relatedWorkSet>
```

Comment:
I don't see an e-shop link related work when running it.

https://harvester-bl.britishart.yale.edu/oaicatmuseum/OAIHandler?verb=GetRecord&identifier=oai:tms.ycba.yale.edu:34&metadataPrefix=lido